packages/api/src/schema/schema.yaml

```diff
@@ -213,6 +213,7 @@ components:
               - multistream.error
               - multistream.disconnected
               - playback.user.new
+              - playback.accessControl
```

Comment:
I noticed that the other event types are named with all lowercase - is kebab case preferred here? Purely stylistic choice, so no strong opinion either way.

Reply:
Yeah, probably worth it too make it lowercase here as well, so `playback.accesscontrol` , I agree!

Reply:
After discussing with Victor, we are keeping the casing here instead of the lower case, if it's fine for you @0xcadams 
would be strange to have a name-space like `access.control` and it's the only one not lowercase because it's the first one with 2 words (user.new is going to be removed soon either way)

Reply:
Sounds great!